Makefile

```diff
@@ -244,9 +244,23 @@ test-local:
 
 .PHONY: test-e2e
 test-e2e: ## Runs all Thanos e2e docker-based e2e tests from test/e2e. Required access to docker daemon.
-test-e2e: docker
+test-e2e: docker-multi-stage
```

Comment:
why multi stage?

Reply:
I can revert this. I thought since this action meant to run on local machine, this would be more convenient because it's platform-independent. I can remove this if it's something you don't want. WDYT?